stellarphot/settings/tests/test_models.py

```diff
@@ -34,6 +36,17 @@
     SourceLocationSettings,
 )
 
+# NOTE: ALWAYS USE DEEPCOPY ON THESE DICTS SO TESTS DO NOT MODIFY THEM
+TEST_CAMERA_VALUES = deepcopy(TEST_CAMERA_VALUES)
+TEST_APERTURE_SETTINGS = deepcopy(TEST_APERTURE_SETTINGS)
+TEST_EXOPLANET_SETTINGS = deepcopy(TEST_EXOPLANET_SETTINGS)
+TEST_OBSERVATORY_SETTINGS = deepcopy(TEST_OBSERVATORY_SETTINGS)
+TEST_PHOTOMETRY_OPTIONS = deepcopy(TEST_PHOTOMETRY_OPTIONS)
+TEST_PASSBAND_MAP = deepcopy(TEST_PASSBAND_MAP)
+TEST_PHOTOMETRY_SETTINGS = deepcopy(TEST_PHOTOMETRY_SETTINGS)
+TEST_LOGGING_SETTINGS = deepcopy(TEST_LOGGING_SETTINGS)
+TEST_SOURCE_LOCATION_SETTINGS = deepcopy(TEST_SOURCE_LOCATION_SETTINGS)
```

Comment:
I have to admit, looking at this, it looks like you are deep copying the originals but to the same variables.  I hadn't thought about this, but it makes sense that deep copy() will generate a new object that you assign to the old variable name... nice way to keep the old code working and still not change the constants.